integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -164,10 +173,28 @@ class AuthorizationServerIT extends ITSupport {
 
         assertThat(createdAuthorizationServer, notNullValue())
 
-        Policy createdPolicy = createdAuthorizationServer.createPolicy(policy)
-        registerForCleanup(createdPolicy)
+        AuthorizationServerPolicy createdAuthorizationServerPolicy = createdAuthorizationServer.createPolicy(authorizationServerPolicy)
 
-        assertPresent(createdAuthorizationServer.listPolicies(), createdPolicy)
+        AuthorizationServerPolicyRule authorizationServerPolicyRule = createdAuthorizationServerPolicy.createPolicyRule(createdAuthorizationServer.getId(),
```

Comment:
TODO: Write more ITs for crud operations on the new `AuthorizationServerPolicy***` models.

Reply:
This is done by @sergiishamrai-okta in https://github.com/okta/okta-sdk-java/pull/475/commits/fedff9b3f7f1b079e5740a0e1316f538acae8c59. Thanks @sergiishamrai-okta